src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -868,12 +868,14 @@ class ContentSidebar extends PureComponent<Props, State> {
      *
      * @private
      * @param {string} commentId - The comment's id
+     * @param {Object} permissions - The comment's permissions
      * @param {Function} successCallback - the function which will be called on success
      * @param {Function} errorCallback - the function which will be called on error
      * @return {void}
      */
     deleteComment = (
         commentId: string,
+        permissions: Object,
```

Comment:
Are tasks going to be in a separate PR? Also even if we aren't going to use editComment right now we should pass in the permissions object.

Reply:
Tasks will be a separate PR, after deliberation with @pramodsum.

I'll update `editComment`